akvo/rsr/tests/views/test_account.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+
+"""
+Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the Akvo RSR module.
+For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+from __future__ import print_function
+
+import json
+
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test import TestCase, Client
+
+from akvo.rsr.models import Employment, Organisation, Partnership, Project, User
+from akvo.utils import check_auth_groups
+
+
+class AccountTestCase(TestCase):
+    """Test account views."""
+
+    def setUp(self):
+        check_auth_groups(settings.REQUIRED_AUTH_GROUPS)
+        self.admin_group = Group.objects.get(name='Admins')
+        self.user_group = Group.objects.get(name='Users')
+        self.username = 'user@example.com'
+        self.password = 'password'
+        self.org1 = Organisation.objects.create(name='akvo', long_name='akvo foundation')
+        self.org2 = Organisation.objects.create(name='icco', long_name='icco foundation')
+        for org in (self.org1, self.org2):
+            project = Project.objects.create()
+            project.publishingstatus.status = 'published'
+            project.publishingstatus.save()
+            Partnership.objects.create(project=project, organisation=org)
+        self.c = Client(HTTP_HOST=settings.RSR_DOMAIN)
+
+    def tearDown(self):
```

Comment:
`check_auth_groups` actually checks if groups exist before trying to create them. So, it doesn't really matter, but I guess for consistency, it's better to delete them. I'll push a fix.